pandas/tests/io/test_common.py

```diff
@@ -222,7 +222,7 @@ def test_read_expands_user_home_dir(
             (pd.read_sas, "os", ("io", "sas", "data", "test1.sas7bdat")),
             (pd.read_json, "os", ("io", "json", "data", "tsframe_v012.json")),
             (pd.read_msgpack, "os", ("io", "msgpack", "data", "frame.mp")),
-            (pd.read_pickle, "os", ("io", "data", "categorical_0_14_1.pickle")),
+            (pd.read_pickle, "os", ("io", "data", "categorical.pickle")),
```

Comment:
should there be a version number on this?

Reply:
Yeah, that will  probably be easier for future maintenance. Will add it.